src/detect.h

```diff
@@ -872,7 +872,7 @@ typedef struct DetectEngineCtx_ {
 
     uint8_t flags;        /**< only DE_QUIET */
     uint8_t mpm_matcher;  /**< mpm matcher this ctx uses */
-    uint16_t spm_matcher; /**< spm matcher this ctx uses */
+    uint8_t spm_matcher;  /**< spm matcher this ctx uses */
```

Comment:
Is this to win some space here ?

IIRC, I had a commit like that which did not make it into master...

Reply:
yeah in another commit I added another int and I didn't want to grow the struct

Reply:
Do you have a warning somehow when a struct gets bigger ? (or is it manual analysis)

Reply:
manual (`pahole`)